AWSIoT/AWSIoTDataManager.m

```diff
@@ -328,7 +322,7 @@ - (instancetype)initWithConfiguration:(AWSServiceConfiguration *)configuration w
         if(_mqttClient == nil){
             AWSDDLogError(@"**** mqttClient is nil. **** ");
         }
-        _mqttClient.userMetaData = [NSString stringWithFormat:@"%@%@", @"?SDK=iOS&Version=", SDK_VERSION];
```

Comment:
Remove SDK_VERSION if it is not being used

Reply:
Please check the latest update; it's already removed.